features/support/ui/your_contributions.rb

```diff
@@ -8,6 +8,11 @@ class YourContributions < SitePrism::Page
     set_url '{/locale}/tools/wpcc/your_contributions'
     set_url_matcher(/wpcc\/\d+\/your_contributions/)
 
+    section :employee_percent,   EmployeePercentSection, '.contributions__source--employee'
+    section :employer_percent,   EmployerPercentSection, '.contributions__source--employer'
+    
     element :details, '.details__row'
+    element :contributions_description, '.t-contributions_description'
```

Comment:
Its proving difficult to target the particular p tag as all browsers dont recognise first-child selector. Using nth-child didnt work either. 

There is also a balance between communicable code, which the t-class does, vs adding unnecessary code.

In the interest of moving forward, I'm going to merge because the markup isn't finalised at this stage and test objects may be further affected